css/v202209/symbiota/header.css

```diff
@@ -152,9 +154,11 @@ header a:hover {
   display: none;
 }
 
-ul.menu {
+ul.menu,
+ul.menu ul {
   list-style: none;
-  padding-left: 0;
+  padding: 0.25em 1em;
```

Comment:
A couple of places are using em instead of rem. How easy would that those be to convert to rem?